src/common/database.c

```diff
@@ -2551,6 +2551,38 @@ static int _upgrade_library_schema_step(dt_database_t *db, int version)
 
     new_version = 44;
   }
+  else if(version == 44)
+  {
+    sqlite3_exec(db->handle, "PRAGMA foreign_keys = OFF", NULL, NULL, NULL);
+    sqlite3_exec(db->handle, "BEGIN TRANSACTION", NULL, NULL, NULL);
+
+    TRY_EXEC("ALTER TABLE main.images ADD COLUMN thumb_timestamp INTEGER default -1",
+             "[init] can't add thumb_timestamp column\n");
+    TRY_EXEC("ALTER TABLE main.images ADD COLUMN thumb_maxmip INTEGER default 0",
+             "[init] can't add thumb_maxmip column\n");
+
+    TRY_EXEC("CREATE TABLE history_hash_new"
+              " (imgid INTEGER PRIMARY KEY,"
+              "  basic_hash BLOB, auto_hash BLOB, current_hash BLOB, mipmap_hash BLOB,"
+              "  FOREIGN KEY(imgid) REFERENCES images(id) ON UPDATE CASCADE ON DELETE CASCADE)",
+            "[init] can't create table history_hash_new\n");
+
+    TRY_EXEC("INSERT INTO history_hash_new"
+             " SELECT imgid, basic_hash BLOB, auto_hash BLOB, current_hash BLOB, mipmap_hash BLOB"
```

Comment:
`BLOB` should not be in the SELECT statement.

Reply:
I hope this is the issue reported above as I have no other idea about the `view v_images: no such column: cm.name` error message !

Reply:
See comment in code - i had to look though some SQLite docs which was pretty new for me. We can't rename a table under certain conditions so i had to use a workaround.